src/backend/monomorph.md

```diff
@@ -55,6 +55,17 @@ The monomorphization collector is run just before MIR lowering and codegen.
 collection and then partitions them into [codegen
 units](../appendix/glossary.md#codegen-unit).
 
+Consider Crate A and Crate B, such that Crate B depends on Crate A.
```

Comment:
It might be good to add a header here for "Codegen Unit Partitioning" or similar. This isn't really part of the "Collection" step anymore (in my mind anyway). 

If you want to write a sentence or two introducing that topic, that would would be awesome, otherwise we can add more content here later.